cypress/e2e/circulation-log/loan-date-and-time.cy.js

```diff
@@ -0,0 +1,110 @@
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import UserEdit from '../../support/fragments/users/userEdit';
+import Checkout from '../../support/fragments/checkout/checkout';
+import TopMenu from '../../support/fragments/topMenu';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import CheckinActions from '../../support/fragments/check-in-actions/checkInActions';
+import { Locations } from '../../support/fragments/settings/tenant/location-setup';
+import Users from '../../support/fragments/users/users';
+import { MultiColumnListCell } from '../../../interactors';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import SearchResults from '../../support/fragments/circulation-log/searchResults';
+import LoansPage from '../../support/fragments/loans/loansPage';
+
+describe('Loan date and time', () => {
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances(),
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    requestsId: '',
+  };
+  let userData;
+  let ITEM_BARCODE;
+
+  before('create inventory instance', () => {
+    cy.getAdminToken();
+    ServicePoints.createViaApi(testData.servicePoint);
+    testData.defaultLocation = Location.getDefaultLocation(testData.servicePoint.id);
+    Location.createViaApi(testData.defaultLocation).then((location) => {
+      InventoryInstances.createFolioInstancesViaApi({
+        folioInstances: testData.folioInstances,
+        location,
+      });
+    });
+    cy.createTempUser([
+      Permissions.circulationLogAll.gui,
+      Permissions.uiInventoryViewInstances.gui,
+      Permissions.uiUsersViewLoans.gui,
+      Permissions.inventoryAll.gui,
```

Comment:
do you need inventory all permissions?

Reply:
add it to the permissions.js 
internal: 'ui-inventory.item.delete',  gui: 'Inventory: View, create, edit, delete items'

Reply:
done